daemon/api_test.go

```diff
@@ -214,15 +189,15 @@ func (s *apiSuite) TestSnapInfoOneIntegration(c *check.C) {
 	s.vars = map[string]string{"name": "foo"}
 
 	// the store tells us about v2
-	s.parts = []*store.RemoteSnap{
-		&store.RemoteSnap{
-			Pkg: remote.Snap{
-				Name:         "foo",
-				Version:      "v2",
-				Description:  "description",
-				Developer:    "bar",
+	s.rsnaps = []*snap.Info{
+		&snap.Info{
```

Comment:
This can be just `[]*snapInfo{{`.. the magic type inference mechanism works well here.